samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResultFromSectionActivity.jsp

```diff
@@ -1,6 +1,7 @@
 <%@ page contentType="text/html;charset=utf-8" pageEncoding="utf-8" %>
 <%@ taglib uri="http://java.sun.com/jsf/html" prefix="h" %>
 <%@ taglib uri="http://java.sun.com/jsf/core" prefix="f" %>
+<%@ taglib uri="http://myfaces.apache.org/tomahawk" prefix="t"%>
```

Comment:
As "gradeStudentResult.jsp" and "gradeStudentResultFromSectionActivity" are the same, I copied all "gradeStudentResult" file to make it consistent and with the same style.
But now I've removed the "gradeStudentResultFromSectionActivity.jsp" to avoid replicated code. Maybe the h1/h2/h3 fix and tomahawk tag removal should be done in a new Jira?